sources/packages/web/src/components/common/LocationSummary.vue

```diff
@@ -108,17 +111,34 @@ export default {
     const formatter = useFormatters();
     const router = useRouter();
     const clientType = computed(() => AuthService.shared.authClientType);
+    const isInstitutionUser = computed(() => {
```

Comment:
`isInstitutionUser` and `isAESTUser` do not need to be computed or even reactive, they are not used in the template or event returned on the setup.
That said, please create the specific logic for the specific client outside the component as much as possible.
Instead of doing the `router.push` inside the component, trigger an event with the `locationId` and do the routing in the component consumer Institution/AEST.

Reply:
Agreed. We can pass the RouteLocationRaw from the parent, unless it is very hard to modify the parent component.